EIPS/eip-8888.md

```diff
@@ -0,0 +1,71 @@
+---
+eip: 8888
```

Comment:
```suggestion
eip: to be assigned
```

Reply:
Thank you very much for your guidance. I acknowledge my limited experience, and I will carefully review my documents again.